lib/cache-control.js

```diff
@@ -0,0 +1,88 @@
+'use strict'
+
+// Cache control header utilities, for more info see:
+// https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control
+// Useful reads:
+// - https://odino.org/http-cache-101-scaling-the-web/
+// - https://web.dev/stale-while-revalidate/
+// - https://csswizardry.com/2019/03/cache-control-for-civilians/
+// - https://jakearchibald.com/2016/caching-best-practices/
+
+const assert = require('assert')
+const ms = require('ms')
+
+const validSingletimes = [
+  'must-revalidate',
+  'no-cache',
+  'no-store',
+  'no-transform',
+  'public',
+  'private',
+  'proxy-revalidate',
+  'immutable'
+]
+
+const validMultitimes = [
+  'max-age',
+  's-maxage',
+  'stale-while-revalidate',
+  'stale-if-error'
+]
+
+function cacheControl (type, time) {
+  const previoustime = this.getHeader('Cache-Control')
+  if (time == null) {
+    assert(validSingletimes.indexOf(type) !== -1, `Invalid Cache Control type: ${type}`)
+    this.header('Cache-Control', previoustime ? `${previoustime}, ${type}` : type)
+  } else {
+    if (typeof time === 'string') {
+      time = ms(time) / 1000
+    }
+    assert(validMultitimes.indexOf(type) !== -1, `Invalid Cache Control type: ${type}`)
+    assert(typeof time === 'number', 'The cache control time should be a number')
+    this.header('Cache-Control', previoustime ? `${previoustime}, ${type}=${time}` : `${type}=${time}`)
+  }
+  return this
+}
+
+function preventCache () {
+  this.header('Cache-Control', 'no-store, max-age=0, private')
```

Comment:
It would be better to set `pragma` and `expires` headers to give compatibility support for HTTP/1.0? @delvedor 

```js
this
  .header('Cache-Control', 'no-store, max-age=0, private')
  .header('pragma', 'no-cache')
  .header('expires', 0)
```

Reply:
see: https://owasp.org/www-community/OWASP_Application_Security_FAQ#how-do-i-ensure-that-sensitive-pages-are-not-cached-on-the-users-browser

Reply:
@mcollina I see. I just wanted to get your opinions in the first place. I am gonna do that.